lib/data_update_scripts/20200916202343_backfill_co_author_ids_for_articles.rb

```diff
@@ -0,0 +1,8 @@
+module DataUpdateScripts
+  class BackfillCoAuthorIdsForArticles
+    def run
+      articles = Article.where.not(second_user_id: nil).or(Article.where.not(third_user_id: nil))
+      articles.update_all("co_author_ids = array[second_user_id, third_user_id]")
```

Comment:
There's a problem in this script:

- load the console
- take an article, set `.second_user_id` to something and save it
- run the two lines here above manually
- reload the article, the result will be something like:

```ruby
[6] pry(main)> a.second_user_id
=> 3
[7] pry(main)> a.co_author_ids
=> [3, nil]
```

I don't think we want nil in that array. 

I think we should iterate on the result of line 4 and build the array in Ruby, remove the nils with `.compact` and then save it to the column with `update_columns(co_author_ids: array)` so to skip callbacks. There might be a way to use SQL but I wound't bother as it's only 59 articles on DEV: https://dev.to/admin/blazer/queries/247-articles-with-coauthors